app/dao/services_dao.py

```diff
@@ -393,6 +395,18 @@ def _delete(query):
             db.session.delete(user)
 
 
+def delete_service_created_for_functional_testing(service):
```

Comment:
Is there anything which prevents this being used on production and/or a live service?

Reply:
This check is done at the top of the request handler, rather than in the DAO code:
![image](https://github.com/user-attachments/assets/963c41de-ab30-4ca5-80ad-77fee408b0da)